models/webHook.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+// WebHook from media4u
+type WebHook struct {
+	Mobilenumber      string
```

Comment:
Why the attribute of the structs are pascal case? Can you make it camel case?

Reply:
I can. If the casing is off, it's due to this being copied from the expected incoming keys from media4u.

Reply:
I mean, sorry my bad. for example `Replyid` can we make it `ReplyId`?